packages/nuxt/src/app/plugins/view-transitions.client.ts

```diff
@@ -67,6 +67,12 @@ export default defineNuxtPlugin((nuxtApp) => {
     resetTransitionState()
   })
 
+  nuxtApp.hook('app:error', () => {
+    // Finish the transition instead of aborting to allow smooth animation to error page
+    finishTransition?.()
```

Comment:
should we do:

```suggestion
    abortTransition?.()
```

...or, alternatively, should we call `finishTransition` within `vue:error` as well?